### PR TITLE
Add arcade to repo

### PR DESCRIPTION
**Pull request by BrennanConroy**
99% build goop.

SDL will run for internal builds now (verified manually).

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <PreReleaseVersionLabel>prerelease</PreReleaseVersionLabel>
```
Comment (JunTaoLuo) [inline eng/Versions.props]: This is a rather unusual prerelease label. Usually we use alpha or preview instead.

Comment (anurse) [reply eng/Versions.props]: Let's just use `preview1`. I think we won't be ready to call it "shipped" until Feb anyway.

**Comment by anurse** [comment]
For a context reminder: We don't ship any artifacts from the build of this repo. The repo itself is the artifact. We will "ship" by tagging the repo and making an update to https://github.com/Microsoft/vcpkg to reference the new commit hash.

File: eng/Versions.props (hunk 1 above)
Comment (BrennanConroy) [inline eng/Versions.props]:
```suggestion
    <PreReleaseVersionLabel>preview1</PreReleaseVersionLabel>
```

**Pull request merged**